roles/backend_setup/tasks/lvm_exclude_filter.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Backup lvm.conf file
+  copy:
+    src: /etc/lvm/lvm.conf
+    dest: /backup
```

Comment:
Will this directory be available? isn't it better to backup to same location with different name?

Reply:
okay,

Reply:
This part of the code is not required. This information is added in the doc, as the user is prone to make such mistakes while editing manually, as this is covered in ansible, this code block is safe to be ignored

Reply:
@satheesaran , meaning backup is not necessary?

Reply:
done